ha/test/integration/health_monitor/test_publisher.py

```diff
@@ -45,10 +45,9 @@ def receive(message):
         state = "failed"
         message_type = event_manager.subscribe('csm', [SubscribeEvent(resource_type, [state])])
         print(f"Subscribed {component}, message type is {message_type}")
-        health_event = HealthEvent("event_1", "failed", "fault", "site_1", "rack_1", "cluster_1", "storageset_1",
-                        "node_1", "abcd.com", "node:fru:disk", "16215009572", "disk_1", None)
+        health_event = HealthEvent("source_1", "event_1", "failed", "fault", "site_1", "rack_1", "cluster_1", "storageset_1", "node_1", "abcd.com", "node:fru:disk", "16215009572", "disk_1", None)
         action_event = RecoveryActionEvent(health_event)
-        event_manager.publish(action_event)
+        event_manager.publish(action_event.event)
```

Comment:
you can use action_event.get_event() here to keep test in sync with code

Reply:
fixed in both files test_publisher and test_k8s_publisher